lib/videoroom/room.ex

```diff
@@ -179,22 +161,15 @@ defmodule Videoroom.Room do
     Membrane.Logger.error("Endpoint #{inspect(endpoint_id)} has crashed!")
     peer_channel = state.peer_channels[endpoint_id]
 
-    error_message = "WebRTC endpoint has crashed, please refresh the page to reconnect"
-
-    data =
-      error_message
-      |> MediaEvent.create_error_event()
-      |> MediaEvent.encode()
-
-    send(peer_channel, {:media_event, data})
+    GenServer.stop(peer_channel, :endpoint_crashed)
```

Comment:
Does it mean that we won't inform FE about endpoint crash? The only thing FE will notice is that WS connection was closed :thinking:

Reply:
Yes. But it should now that it didn't close cleanly. Also, `error` media event doesn't exist anymore https://github.com/jellyfish-dev/membrane_rtc_engine/pull/214#discussion_r1062436369. But we probably need to make sure that the FE handles phoenix channel errors correctly